### PR TITLE
Allow specifying netns name for StandaloneNetworkNamespace

### PR DESCRIPTION
**Pull request by ctan-g**
This commit allows specifying `netns` name for `StandaloneNetworkNamespace` for static style use.

### CODE REVIEW

File: silk/device/netns_base.py [hunk 1]
```diff
@@ -190,6 +190,14 @@ def add_route(self, dest, dest_subnet_length, via_addr, interface_name):
                                                            via_addr, interface_name)
         self.make_netns_call_async(command, "", 1, None)
 
+    def log_debug(self, log_line: str):
```
Comment (qilinkang) [inline silk/device/netns_base.py]: Should we create instance of class StandaloneNetworkNamespace(netns_name) which implements all logs ?

Comment (ctan-g) [reply silk/device/netns_base.py]: Right, this makes the most sense.

**Review by qilinkang — APPROVED** [review]
LGTM

**Review by priyankaCh22 — APPROVED** [review]
LGTM

File: .codecov.yml [hunk 2]
```diff
@@ -6,4 +6,4 @@ coverage:
 
     patch:
       default:
-        target: 10%
+        target: 0%
```
Comment (jwhui) [inline .codecov.yml:9]: Is this change intentional?

Comment (ctan-g) [reply .codecov.yml:9]: Yes, since we haven't covered these util classes yet, I discussed with @qilinkang and temporarily set it to zero to allow the checks to pass.

**Review by jwhui — APPROVED** [review]
